src/screens/Sleep.tsx

```diff
@@ -1,9 +1,144 @@
 import React from 'react';
-import { View } from 'react-native';
+import {View, Text, StyleSheet, TouchableOpacity, Alert,} from 'react-native';
```

Comment:
Also, there should be spaces after the open curly bracket and before the closing curly bracket.
```suggestion
import { View, Text, StyleSheet, TouchableOpacity, Alert } from 'react-native'
```

Reply:
This should be done for all the import statements.